### DISCUSSION TITLE
Fields removed during update process

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#14492] [General] [answered]
@mariuszkrzaczkowski 

After update to 6.1 I has notices what Sales Orders field Company name was removed from system including database column. This means what all data my removes and I can't generate my old Single Orders

Pls don't remove fields in future! Make them inactive.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
attach field names are tests, update should not delete

**Reply by vovpff** [reply]
For this time I found only one field:
Company name (company) in Sales Orders

dump from my 5.3

```
CREATE TABLE `u_yf_ssingleorders` (
...
  `company` varchar(255) DEFAULT NULL,
...
```

**Reply by rskrzypczak** [reply]
The field should not be deleted if it is in use.
see: https://github.com/YetiForceCompany/UpdatePackages/blob/developer/YetiForce%20CRM%205.x.x/5.3.0_to_6.0.0/updates/init.php#L1105-L1136

**Reply by vovpff** [reply]
This is fact what field is removed. Also this field unavailable on gitdev.
